src/opnsense/mvc/app/controllers/OPNsense/TrafficShaper/forms/dialogPipe.xml

```diff
@@ -1,31 +1,31 @@
 <form>
     <field>
         <id>pipe.enabled</id>
-        <label>enabled</label>
+        <label>Enabled</label>
         <type>checkbox</type>
-        <help>enable this pipe and it's related queues and rules</help>
+        <help>Enable this pipe and it's related queues and rules</help>
     </field>
     <field>
         <id>pipe.bandwidth</id>
-        <label>bandwidth</label>
+        <label>Bandwidth</label>
         <type>text</type>
         <help>Total bandwidth for this pipe</help>
     </field>
     <field>
         <id>pipe.bandwidthMetric</id>
-        <label>bandwidth Metric</label>
+        <label>Bandwidth Metric</label>
```

Comment:
Can we agree on first letter uppercase, all else normal lowercase unless such things like "IP" or "IPsec" or "OpenVPN" or "CoDel" ?

Reply:
In Plugins all labels begin with upper first on all words I think

Reply:
looks mixed. we need to establish a baseline from core either way

Reply:
(and ideally avoid bouncing perfectly good translations)